tests/requirement/test_requirement_disease_classification_choice_is.py

```
@@ -0,0 +1,97 @@
+\
+from django.test import TestCase
+from logging import getLogger
+
+from endoreg_db.models import (
+    Requirement, 
+    PatientDisease,
+    Disease,
+    DiseaseClassificationChoice, # Added DiseaseClassificationChoice
```

Ruff (0.11.9)

`endoreg_db.models.DiseaseClassificationChoice` imported but unused

Remove unused import

(F401)